src/transformer.cpp

```diff
@@ -191,6 +191,9 @@ std::unique_ptr<BlockComment> Transformer::visit(
 
 std::unique_ptr<InlineVerilog> Transformer::visit(
     std::unique_ptr<InlineVerilog> node) {
+  for (auto&& symbol : node->interpolated_symbols) {
```

Comment:
From my understanding of this code, it seems like you're basically relying on the fact that a transformer always transforms an expression in the same way (i.e. it is a "pure function", at least for this subset of expressions) and apply the transformation to the expression to be interpolated.

Presumably this works if we only have simple transformers and simple expressions, but this seems pretty fragile to me, both (a) for the reason that transformers can do very general things, and (b) because the referenced expression may not even exist post-transformation.

The most robust thing to do seems like keeping another symbol table that is updated by all transformations. Or providing a unified way for each transformation to maintain that table.

Out of curiosity, how would the following work:
```python
add add_inst(...);
assert property (... add_inst.O ...);
```
and then `add_inst` gets inlined into `x + y`? Are interpolated symbols restricted to expressions only?